app/routes/api.js

```diff
@@ -314,4 +314,16 @@ apiRouter.get("/discounts/:code", (req, res) => {
     query("SELECT discount FROM discounts WHERE code = $1", [code], res);
 });
 
+apiRouter.put("/orders/process/:id", (req, res) => {
+    const id = req.params.id;
+    const {discountAmount, tipAmount, total, isPaid} = req.body;
+
+    query(
+        "UPDATE orders SET discount = $1, tips = $2, total = $3, is_paid = $4 WHERE id = $5",
```

Comment:
you can simply do `is_paid = true` for this and remove the isPaid from the post request and values

Reply:
Hmm I can't really think of another scenario in where we use this endpoint, so I'll manually write this into the query string for the time being.